content/hi/docs/reference/glossary/affinity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: आत्मीयता
+id: affinity
+date: 2019-01-11
+full_link: /docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+short_description: >
+     पॉड्स को कहां रखा जाए, यह निर्धारित करने के लिए शेड्यूलर द्वारा उपयोग किए जाने वाले नियम
+aka:
+tags:
+- fundamental
+---
+
+कुबेरनेट्स में, _affinity_ नियमों का एक सेट है जो शेड्यूलर को संकेत देता है कि पॉड्स को कहाँ रखा जाए।
```

Comment:
I feel that instead of transliterating `sets` we should localize it. WDYT @khannakshat7 @anubha-v-ardhan ?
```suggestion
कुबेरनेट्स में, _affinity_ नियमों का एक संग्रह है जो शेड्यूलर को संकेत देता है कि पॉड्स को कहाँ रखा जाए।
```

Reply:
Instead of using `संग्रह` we could also use `समूह`. Thoughts? Thanks

Reply:
Thanks @Babapool, समूह suits better in this scenario.